lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -864,12 +868,31 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
             binder.getLoc(), rewriter.getType<Torch::FloatType>(),
             rewriter.getFloatAttr(rewriter.getF64Type(), gamma));
 
-        Value vInputScale = rewriter.create<Torch::ConstantFloatOp>(
+        Value cstOne = rewriter.create<Torch::ConstantFloatOp>(
             binder.getLoc(), rewriter.getType<Torch::FloatType>(),
             rewriter.getFloatAttr(rewriter.getF64Type(), 1.0));
 
-        rewriter.replaceOpWithNewOp<Torch::AtenEluOp>(
```

Comment:
what's wrong with just AtenEluOp and relying on DecomposeComplexOps to do the following 20 ish lines? does it cause a failure sohewhere?

Reply:
AtenEluOp is not an equivalent of Selu Op.

Reply:
yes, but it looks like we can just add a scaling factor to Elu to get Selu. This way we have a little less code duplication.

Reply:
I have already tried that implementation but it's not working maybe because of the way Elu is lowered in Torch-MLIR.

Reply:
ah gotcha. thanks for letting me know.